src/ansible_navigator/_version_doc_cache.py

```diff
@@ -0,0 +1,14 @@
+"""Version information for ansible-navigator collection doc cache.
+
+.. note::
+
+   __version_collection_doc_cache__:
+
+   Indicates the version of the schema of the collection doc cache
+   this is checked during initialization, if the version of the cache
+   differs from below, the cache will be rebuilt.  This should be
+   incremented when the schema changes and need not correspond to the
+   application version, although keeping the major in sync is probably
+   not a bad idea to minimize the amount of stale docs in the user's cache
+"""
+__version_collection_doc_cache__ = "1.0"
```

Comment:
Maybe these docstrings should be separate?
```suggestion
"""Version information for ansible-navigator collection doc cache."""

__version_collection_doc_cache__ = "1.0"
"""
Indicates the version of the schema of the collection doc cache
this is checked during initialization, if the version of the cache
differs from below, the cache will be rebuilt.  This should be
incremented when the schema changes and need not correspond to the
application version, although keeping the major in sync is probably
not a bad idea to minimize the amount of stale docs in the user's cache
"""
```

Reply:
I think it's fine as is.